app/safe/internal/server/route/list.go

```diff
@@ -22,7 +22,9 @@ import (
 	"strings"
 )
 
-func List(cid string, w http.ResponseWriter, r *http.Request, spiffeid string) {
+func doList(cid string, w http.ResponseWriter, r *http.Request,
```

Comment:
both list and listencrypted functions had a lot of commonalities, so moved the common parts here.